api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -97,6 +80,21 @@
 import org.openmrs.util.OpenmrsConstants;
 import org.openmrs.util.PrivilegeConstants;
 
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.openmrs.test.OpenmrsMatchers.hasId;
+import static org.openmrs.test.TestUtil.containsId;
+
 /**
```

Comment:
I guess this file was touched accidentally. Correct?

Reply:
Yes, the order of imports changed and some formatting, I probably included it when applying the OpenMRS formatting conventions. Should I revert the changes?

Reply:
If it does not take too much of your time.

Reply:
While reverting the changes I realised that one change is needed, namely to make the inner class `SomeTestOrder` public so that it is visible to `ValidateHibernateMappingsDatabaseIT`.